### ISSUE TITLE
Feature request - key modifier to alter "show windows from all workspaces"?

### ISSUE THREAD

**Issue by bowars** [klangman/CassiaWindowList#22]
Another idea - how about being able to hold a particular modifier (shift or control) to switch off (or on) "include windows from all workspaces" on-the-fly?

**Comment by klangman** [comment]
Good idea.. Let me think about it for a bit.. Are you thinking that hitting the modifier key would show windows from other workspaces but only while holding the key, or more like a toggle?

**Comment by klangman** [comment]
OK.. So here is the problem with this idea for which I have not thought of any clean way to avoid. 

There already exists the options to setup mouse click actions which occur when clicking on a window list button while holding the Ctrl or Shift modifier keys (i.e. Shift-Click to open a new application window).  Typically people will hit the modifier and then move the mouse to the window-list to click on a button. If the modifier also had action that you are proposing, then the contents of the window-list would change and it would be a bit awkward. Particularly if it was implemented the why I was thinking where the enter event (when the mouse starts hovering the window-list) is used to trigger the change in the visible set of buttons. This would mean that when the user presses Shift and moved the pointer to the window-list, the contents of the list would change a moment before they click to open a new window.

This is where I am stuck right now. I was thinking of using the "Alt" modifier but for some reason there is no "Alt" defined in Clutter.ModifierType which is why the exiting options to set Ctrl/Shift mouse click actions does not include "Alt".

If I can't think of a solution, then it might not be something I can add.

**Comment by bowars** [comment]
According to a quick Google, Alt should be `Clutter.ModifierType.MOD1_MASK`. Otherwise, perhaps ctrl+shift could work here? Or just a binary option to have this behaviour or the other!

**Comment by klangman** [comment]
Good googleing skills. Did a quick google and didn't find anything, but you are right, MOD1_MASK is working for the Alt key. Thanks!
